src/core/server/saved_objects/import/resolve_import_errors.ts

```diff
@@ -81,40 +88,36 @@ export async function resolveSavedObjectsImportErrors({
   }
 
   // Validate references
-  const { filteredObjects, errors: validationErrors } = await validateReferences(
+  const { filteredObjects, errors: referenceErrors } = await validateReferences(
     objectsToResolve,
     savedObjectsClient,
     namespace
   );
-  errorAccumulator = [...errorAccumulator, ...validationErrors];
+  errorAccumulator = [...errorAccumulator, ...referenceErrors];
+
+  // Check multi-namespace object types for regular conflicts and ambiguous conflicts
+  const importIdMap = await getImportIdMapForRetries(filteredObjects, { typeRegistry, retries });
```

Comment:
We can remove the `await` here since `getImportIdMapForRetries` doesn't need to be async